### DISCUSSION TITLE
Do you have an ETA to support ElastiCache on Spring Cloud AWS 3.x?

### DISCUSSION THREAD

**Discussion by efumas** [awspring/spring-cloud-aws#801] [General]
We would like to know if on Spring Cloud AWS 3.x you will add support for the ElastiCache service.

Thanks

**Comment by maciejwalkowiak** [comment]
Are you looking for Memcached or Redis support? We did not add Elasticache support in 3.0 as the built-in support for Redis in Spring Boot is good enough.

**Reply by efumas** [reply]
We are looking for Memcached support

**Reply by maciejwalkowiak** [reply]
Maybe this will help: https://github.com/sixhours-team/memcached-spring-boot

If you're willing to contribute memcached support to Spring Cloud AWS we will think about it again
